onnxruntime/core/providers/cpu/ml/ml_common.h

```diff
@@ -322,6 +322,7 @@ void write_scores(std::vector<T>& scores, POST_EVAL_TRANSFORM post_transform, in
           scores[0] = 1.f - scores[0];
           break;
         case 2:  //2 = mixed weights, winning class is positive
+        case 3:  //3 = mixed weights, winning class is negative
           if (post_transform == POST_EVAL_TRANSFORM::LOGISTIC) {
             scores.push_back(ComputeLogistic(scores[0]));  //ml_logit(scores[k]);
             scores[0] = ComputeLogistic(-scores[0]);
```

Comment:
These two could be just one line: 
scores[0] = -1 * ComputeLogistic(-scores[0]);

Reply:
I prefer to leave it that way. It may introduce minor discrepencies.